tests/Storage/Microsoft.Bot.Builder.Azure.Tests.Common/Microsoft.Bot.Builder.Azure.Tests.Common.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework Condition="'$(BuildTarget)' == 'netcoreapp31'">netcoreapp3.1</TargetFramework>
+    <TargetFrameworks Condition="'$(BuildTarget)' == ''">netcoreapp3.1</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <Configurations>Debug;Release</Configurations>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.1.1" />
```

Comment:
Replace this entire item group by 

```
  <ItemGroup>
    <PackageReference Include="xunit.extensibility.core" Version="2.4.1" />
  </ItemGroup>
```

This will change the icon of the project to be a standard class library and not a test project (this is the reference you use to create xunit helper libraries.

Reply:
also had to include xunit, but was able to remove the other